Assets/Scripts/Models/Job/JobManager.cs

```diff
@@ -140,4 +190,10 @@ private void DebugLog(string message, params object[] par)
     {
         UnityDebugger.Debugger.LogFormat("FSM", message, par);
     }
+
+    [System.Diagnostics.Conditional("FSM_DEBUG_LOG")]
+    private void DebugLogError(string message, params object[] par)
+    {
+        UnityDebugger.Debugger.LogErrorFormat("FSM", message, par);
+    }
```

Comment:
I don't think these sorts of functions should be needed anymore, they were only a stopgap before implementing our logger and probably should have been removed once that was done, almost certainly don't need to add any more.

Reply:
This also creates a problem where clicking on the error in the editor doesn't get you to the line where the error was actually printed.

Reply:
Good point, I made a better change in my newest PR.